File: CS/dxSample/MyPivotGridControl.cs
using DevExpress.XtraPivotGrid;
using DevExpress.XtraPivotGrid.Data;

namespace DXSample {
    public class MyPivotGridControl :PivotGridControl {
        public MyPivotGridControl() : base() { }
        protected override PivotGridViewInfoData CreateData() {
            return new MyPivotGridViewInfoData(this);
        }
    }
}

File: VB/dxSample/MyPivotGridControl.vb
Imports DevExpress.XtraPivotGrid
Imports DevExpress.XtraPivotGrid.Data

Namespace dxSample
	Public Class MyPivotGridControl
		Inherits PivotGridControl

		Public Sub New()
			MyBase.New()
		End Sub
		Protected Overrides Function CreateData() As PivotGridViewInfoData
			Return New MyPivotGridViewInfoData(Me)
		End Function
	End Class
End Namespace